src/autocomplete.css

```diff
@@ -152,6 +152,15 @@
   font-weight: 400;
 }
 
+/* Try font "GDS Transport" for GOV.UK form groups */
+.govuk-form-group .autocomplete__hint,
+.govuk-form-group .autocomplete__input,
+.govuk-form-group .autocomplete__option {
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
```

Comment:
To add to what @romaricpascal [was saying](https://github.com/alphagov/accessible-autocomplete/pull/598#issuecomment-2066549418):

This could instead be part of a separate `autocomplete-govuk.css` (or better named) stylesheet that is referenced in `govuk-prototype-kit.config.json`.

That way autocomplete users can choose from:

- GOVUK styling `autocomplete-govuk.css`
- Default styling `autocomplete.css`
- No styling (DIY)

Also wanted to mention: the historic reason for why the autocomplete doesn't use GOVUK Frontend classes is because it precedes v0.1 of GOVUK Frontend. It's also a nice feature which has allowed it to be adopted in places outside Gov such as FT or Wordpress. So yes, keeping the "Default styling" agnostic is still worth doing. 👍

Reply:
I'm wondering if this might work:
```
.govuk-template .autocomplete__hint,
...
```
 - simpler to have one css file
 - one request is better for performance
 - `.govuk-template` is always present in the Prototype Kit, whereas there may not be a parent `.govuk-form-group`